src/main/java/com/passbook/sparkeighteen/peristence/entity/ProfileEntity.java

```diff
@@ -0,0 +1,49 @@
+package com.passbook.sparkeighteen.peristence.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.*;
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+@Setter
+@Getter
+@Builder
+@AllArgsConstructor
```

Comment:
@Ketan-SP18 @Bhavesh-SP18 @Hrishikesh-SP18 @Aashishverma06 
Did you guys understand why I removed @Data and added @Getter and @Setter Please and expplain here and also add the links you are reading.

Reply:
yes 
@Data is a convenient shortcut annotation that bundles the features of @ToString, @EqualsAndHashCode, @Getter / @Setter and @RequiredArgsConstructor together.
If we need to set non-default values for any of these parameters, just add those annotations explicitly;
[Link](https://projectlombok.org/features/Data)